Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerQuoteBubbleView_Tests.swift

```diff
@@ -0,0 +1,141 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+@testable import StreamChat
+import StreamChatTestTools
+@testable import StreamChatUI
+import XCTest
+
+class ChatMessageComposerQuoteBubbleView_Tests: XCTestCase {
+    var view: ChatMessageComposerQuoteBubbleView!
+
+    override func setUp() {
+        super.setUp()
+        view = ChatMessageComposerQuoteBubbleView().withoutAutoresizingMaskConstraints
+        view.addSizeConstraints()
+        view.backgroundColor = .gray
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        view = nil
+    }
+
+    func test_emptyAppearance() {
+        view.content = makeMessage(text: "")
+
+        AssertSnapshot(view, variants: .onlyUserInterfaceStyles)
+    }
+
+    func test_defaultAppearance() {
+        view.content = makeMessage(text: "Hello Vader!")
+
+        AssertSnapshot(view)
+    }
+
+    func test_withImageAttachmentAppearance() {
+        let attachment = ChatMessageDefaultAttachment(imageUrl: TestImages.yoda.url, title: "")
+        view.content = makeMessage(text: "Hello Vader!", attachments: [attachment])
+
+        AssertSnapshot(view)
+    }
+
+    func test_withLongTextAppearance() {
+        let attachment = ChatMessageDefaultAttachment(imageUrl: TestImages.yoda.url, title: "")
+        view.content = makeMessage(text: "Hello Darth Vader! Where is my light saber?", attachments: [attachment])
+
+        AssertSnapshot(view)
+    }
+
+    func test_appearanceCustomization_usingUIConfig() {
+        class TestView: UIView {
+            override init(frame: CGRect) {
+                super.init(frame: frame)
+
+                layer.borderColor = UIColor.yellow.cgColor
+            }
+
+            @available(*, unavailable)
+            required init?(coder: NSCoder) {
+                fatalError("init(coder:) has not been implemented")
+            }
+        }
+
+        var config = UIConfig()
+        config.messageComposer.quotedMessageBubbleSubviews.contentView = TestView.self
+
+        view.uiConfig = config
+
+        AssertSnapshot(view, variants: [.defaultLight])
```

Comment:
@VojtaStavik This does not work properly, the border color will be overridden by the appearance life cycle. Should the views of UIConfig really be the most generic ones? In this case there's actually no option, but maybe a better option is to make the UIConfigs conform to `Customizable`? Or the appearance of the views should always be customized in the appearance life cycle of the Parent Customizable view?